src/Dismiss.php

```diff
@@ -82,7 +82,11 @@ public function add_script() {
 
 		$script = <<<EOD
 jQuery( function() {
-    var dismissBtn  = document.querySelector( '#wptrt-notice-$id .notice-dismiss' );
+    var dismissBtn = document.querySelector( '#wptrt-notice-$id .notice-dismiss' );
+
+    if( ! dismissBtn ) {
```

Comment:
Nitpick: WP Coding Standards 
```suggestion
    if ( ! dismissBtn ) {
```